ems/generators/duration/poisson_duration.py

```diff
@@ -15,16 +15,13 @@
 class PoissonDurationGenerator(DurationGenerator):
 
     def __init__(self,
-                 quantity: int,
-                 duration: float):
-        self.quantity = quantity
-        self.duration = duration
-        self.lmda = quantity / duration
+                 lmda: float):
+        self.lmda = lmda
 
     def generate(self,
-                 ambulance: Ambulance,
-                 destination: Point,
-                 timestamp: datetime):
+                 ambulance: Ambulance = None,
+                 destination: Point = None,
+                 timestamp: datetime = None):
         rand = -math.log(1.0 - random.random())
         minutes_until_next = rand / self.lmda
```

Comment:
This probably needs a better variable name than lmda because it's not immediately clear that this is a divisor for a formula